plugins/katello/nightly/upgrade/clients.md

```diff
@@ -18,9 +18,9 @@ Update the Katello client release packages:
 <p>
   Select your Operating System:
   <select id="operatingSystems">
-     <option value="el5">Enterprise Linux 5 (CentOS, etc.)</option>
-     <option value="el6">Enterprise Linux 6 (CentOS, etc.)</option>
-     <option value="el7">Enterprise Linux 7 (CentOS, etc.)</option>
+     <option value="el5">Enterprise Linux 5 (RHEL5, CentOS, etc.)</option>
+     <option value="el6">Enterprise Linux 6 (RHEL6, CentOS, etc.)</option>
+     <option value="el7">Enterprise Linux 7 (RHEL7, CentOS, etc.)</option>
```

Comment:
I think this could just be `RHEL` (since the 5,6,7 is implied by the previous part)